api/managementpb/mysql.proto

```diff
@@ -24,17 +24,17 @@ import "github.com/mwitkow/go-proto-validators/validator.proto";
 import "google/api/annotations.proto";
 import "inventorypb/agents.proto";
 import "inventorypb/services.proto";
+import "managementpb/node.proto";
 import "protoc-gen-swagger/options/annotations.proto";
 
 // Add MySQL
 
 message AddMySQLRequest {
   // Node identifier on which a service is been running. Required.
-  string node_id = 1 [
-    (validator.field) = {
-      string_not_empty: true
-    }
-  ];
+  // Use only one of these paramse (node_id, node_name or register_node)
+  string node_id = 1;
+  string node_name = 21;
+  RegisterNodeRequest register_node = 22;
```

Comment:
All `XXXRequest` messages must be independent and not try to share common properties for future compatibility. Additionally, `AddMySQLRequest` and `RegisterNodeRequest` already share a field: `address`.

We have to copy&paste fields there.